test/functional/uri_options_spec.test.js

```diff
@@ -10,31 +10,36 @@ const MongoParseError = core.MongoParseError;
 const loadSpecTests = require('../spec').loadSpecTests;
 
 describe('URI Options (spec)', function() {
-  loadSpecTests('uri-options').forEach(suite => {
-    describe(suite.name, () => {
-      suite.tests.forEach(test => {
-        const itFn = test.warning ? it.skip : it;
+  loadSpecTests('uri-options')
+    .filter(suite => {
+      if (suite.name === 'tls-options') return false;
```

Comment:
@mbroadst doe we wanna skip the whole suite here or just tests that include `tlsAllowInvalidCertificates` and `tlsDisableOCSPEndpointCheck`?

Reply:
I would think the latter, we definitely support `tls` options outside of those two.